cypress/e2e/telegrafs.test.ts

```diff
@@ -1,7 +1,7 @@
 import {Organization} from '../../src/types'
 
 // a generous commitment to delivering this page in a loaded state
-const PAGE_LOAD_SLA = 10000
+const PAGE_LOAD_SLA = 20000
```

Comment:
idk why, but sometimes it takes forever for cypress to load a page. it just sits there with the dark grey background... doing nothing